src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -280,12 +295,17 @@ void InputProcessor::processInput()
         if (!DataGlobals::isEpJSON) {
             bool success = true;
             epJSON = idf_parser->decode(input_file, schema, success);
+
             //			bool hasErrors = processErrors();
             //			if ( !success || hasErrors ) {
             //				ShowFatalError( "Errors occurred on processing input file. Preceding condition(s) cause termination." );
             //			}
+
             if (DataGlobals::outputEpJSONConversion) {
-                input_file = epJSON.dump(4, ' ', false, json::error_handler_t::replace);
+                json epJSONClean = epJSON;
+                cleanEPJSON(epJSONClean);
+                input_file = epJSONClean.dump(4, ' ', false, json::error_handler_t::replace);
+                //input_file = epJSON.dump(4, ' ', false, json::error_handler_t::replace);
```

Comment:
No unit test for this new function?  Seems like it would be worth passing in a full epJSON instance and verifying it handles all of it correctly.